contrib/pkginstaller/Makefile

```diff
@@ -45,6 +47,12 @@ package_root: get_gvproxy get_qemu
 pkginstaller: packagedir
 	cd $(PACKAGE_DIR) && ./package.sh ..
 
+_notarize: pkginstaller
+	xcrun notarytool submit --apple-id $(NOTARIZE_USERNAME) --password $(NOTARIZE_PASSWORD) -f json --wait out/$(PKG_ARCHIVE_NAME)
```

Comment:
Alright, I ran into two errors: One: 
```
xcrun notarytool submit --apple-id $(NOTARIZE_USERNAME) --password $(NOTARIZE_PASSWORD) -f json --wait out/$(PKG_ARCHIVE_NAME)`
Error: out must be a zip archive (.zip), flat installer package (.pkg), or UDIF disk image (.dmg)
```

I then re-ran with `PKG_ARCHIVE_NAME=podman-installer-macos-aarch64.pkg` (That seemed to be the pkg name according the to makefile). I'd expect this line to automatically grab PKG_ARCHIVE_NAME and not require me to specify it.

Second, notarytool requires a --team-id flag as well. 
```
xcrun notarytool submit --apple-id $(NOTARIZE_USERNAME) --password $(NOTARIZE_PASSWORD) -f json --wait out/$(PKG_ARCHIVE_NAME)`
Must provide all app-specific password authentication arguments (--apple-id, --password, --team-id). You may leave '--password' unspecified on initial invocation to receive a secure prompt.
```
```suggestion
	xcrun notarytool submit --apple-id $(NOTARIZE_USERNAME) --password $(NOTARIZE_PASSWORD)  --team-id=$(NOTARIZE_TEAM) -f json --wait out/$(PKG_ARCHIVE_NAME)
```

Reply:
Nice, will look at the pkg name issue and update the script as suggested, so it did pass notarisation and apple didn't complain about missing signature on some of the files?